packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,33 @@
+This tutorial will be exploring different ways of rendering a list of
+items in Lit!
+
+## Previous knowledge
+It's recommended to have an understanding of
+[defining a component](/docs/components/defining/),
+[rendering](/docs/components/rendering/),and
+[reactive properties](/docs/components/properties/) before
+working on this tutorial.<br>
+Recommended prerequisite: [Intro to Lit](/tutorials/intro-to-lit/).
+
+
+## Rendering a list
+Begin by rendering a list of names from an array. The example already has set
+up a custom element with the `names` property that is an array of string.
+
+The `map` array prototype method can be utilized to generate a list item
```

Comment:
Def defer to @arthurevans here but I think we want to try to avoid passive voice and be more direct. Perhaps something like "Use Array's `map`  method to generate a list..."

Reply:
Elliott has pointed this out in other places too. Reworded.